proto/streamlit/proto/BackMsg.proto

```diff
@@ -46,18 +46,28 @@ message BackMsg {
     // DEPRECATED. This is now inside ClientState.
     // WidgetStates update_widgets = 9;
 
-    // Set to true to ask the server to close the connection
-    bool close_connection = 10;
+    // DEPRECATED. Set to true to ask the server to close the connection
+    // bool close_connection = 10;
 
     ClientState rerun_script = 11;
 
     bool load_git_info = 12;
+
+    // Test and dev-mode only field used to ask the server to disconnect the
+    // client's websocket connection.
+    bool disconnect_websocket = 14;
+
+    // Test and dev-mode only field used to ask the server to shut down the
+    // runtime.
+    bool shutdown_runtime = 15;
```

Comment:
To prevent potential security heart-attacks, do you mind extending these docstrings to say something like 

```
// This message is IGNORED unless Streamlit Runtime is configured with `global.developmentMode = True`

Reply:
Also, can we prepend some string to these names to indicate they're for debugging? `debug_disconnect_websocket`; `debug_shutdown_runtime`

Reply:
Oh, and to be truly security minded, can we add a couple quick tests that prove these messages *are* ignored if `developmentMode == False`?

Reply:
Done!